packages/webapp/src/util/rruleTranslation/fr.js

```diff
@@ -0,0 +1,48 @@
+const frStrings = {
+  every: 'chaque',
+  until: "jusqu'à",
+  day: 'jour',
+  days: 'jours',
+  week: 'semaine',
+  weeks: 'semaines',
+  on: 'sur',
+  in: 'dans',
+  'on the': 'sur le',
+  for: 'pour',
+  and: 'et',
+  or: 'ou',
+  at: 'à',
+  last: 'dernier',
+  '(~ approximate)': '(~ approximatif)',
+  times: 'fois',
+  time: 'temps',
+  minutes: 'minutes',
+  hours: 'heures',
+  weekdays: 'jours de la semaine',
+  weekday: 'jour de la semaine',
+  months: 'mois',
+  month: 'mois',
+  years: 'ans',
+  year: 'an',
```

Comment:
I don't know! Did Calum respond? We should probably check Spanish with Iván sooner rather than later too.

Reply:
Oh okay, I thought Chat GPT knew those were not necessary! I will wait for Calum to respond :)

Reply:
Update:
Calum did some research for us:

> The main issue is that "1st" maps to "premiere", but  "21st" maps to "vingt et unième", and similarly "31st" maps to "trente et unième". You can see a good list of these here: https://francais.lingolia.com/fr/vocabulaire/nombres-date-et-heure/les-nombres-ordinaux

> So I think this will work, you just might have to work on some edge cases depending on which options you aim to support in the UI

> Although, you may also notice that with the exception of 1st (aka 1er/1re), the rest of the numbers are all of the form <digits>e, where the e is raised (see the table on the link above). I'm not sure how to type/render that into html though, but if you could get that to work then you should be good to go!

> But, from [this source](https://www.btb.termiumplus.gc.ca/tpv2guides/guides/clefsfp/index-fra.html?lang=fra&lettr=indx_catlog_n&page=91agpPDdxblQ.html), you should avoid just adding -ième, -ème, or -ère to the end of the number

The last solution seems good, so I'll set "st", "nd", "rd", "th" to an empty string!